### PR TITLE
add mobx to the state managements approaches list

### PR DESCRIPTION
**Pull request by buncismamen**

### CODE REVIEW

**Comment by buncismamen** [comment]
I signed it!

File: src/docs/development/data-and-backend/state-mgmt/options.md [hunk 1]
```diff
@@ -54,4 +54,9 @@ Learn more at the following links, many of which have been contributed by the Fl
   by Saowen
 * [Architect your Flutter project using BLoC pattern]({{site.medium}}/flutterpub/architecting-your-flutter-project-bd04e144a8f1),
   by Sagar Suri
-* [Bloc Library](https://felangel.github.io/bloc), by Felix Angelov
\ No newline at end of file
+* [Bloc Library](https://felangel.github.io/bloc), by Felix Angelov
+
+## Mobx
```
Comment (pavanpodila) [inline src/docs/development/data-and-backend/state-mgmt/options.md]: MobX should be spelled with the ending 'X' in capital, as in **MobX**

File: src/docs/development/data-and-backend/state-mgmt/options.md [hunk 2]
```diff
@@ -54,4 +54,9 @@ Learn more at the following links, many of which have been contributed by the Fl
...
+
+* [Mobx.dart, Hassle free state-management for your Dart and Flutter apps](https://github.com/mobxjs/mobx.dart)
```
Comment (pavanpodila) [inline src/docs/development/data-and-backend/state-mgmt/options.md]: Same here: **MobX.dart**

File: src/docs/development/data-and-backend/state-mgmt/options.md [hunk 3]
```diff
@@ -54,4 +54,9 @@ Learn more at the following links, many of which have been contributed by the Fl
...
+* [Getting started with mobx.dart](https://mobx.pub/getting-started)
```
Comment (pavanpodila) [inline src/docs/development/data-and-backend/state-mgmt/options.md]: and here...**MobX.dart**

**Comment by sfshaza2** [comment]
@filiph, can you take a look?

**Pull request merged**